UnityProject/Assets/Scripts/Lifecycle/PlayerSpawn.cs

```diff
@@ -41,6 +41,9 @@ public static void ServerSpawnPlayer(JoinedViewer forViewer, Occupation occupati
 			{
 				SecurityRecordsManager.Instance.AddRecord(newPlayer.GetComponent<PlayerScript>(), occupation.JobType);
 			}
+			var Element = PlayerList.GetConnectedPlayer(newPlayer);
+			Element.Job = occupation.JobType;
+			PlayerList.Instance.AddOrUpdate(Element);
```

Comment:
There's a better place to do this without needing the new function. I'll push a commit onto your branch.

Reply:
oh, yeah Keep the function though since that's useful I think